Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Probably shoudl reference `MSBuild.Sdk.Extras` instead of `Microsoft.NET.Sdk`. That adds support for the nesting and settings, resx, etc.

Reply:
Then add a property `<ExtrasEnableWpfProjectSetup>true</ExtrasEnableWpfProjectSetup>`

Reply:
@onovotny Can we take that up in a follow up PR?

Reply:
Sure

Reply:
Can you please create an issue to track that clean up though?

Reply:
Tracked in #2451